src/P3Scheme.jl

```diff
@@ -228,7 +228,7 @@ Returns the shape parameter μ for a given λ value
 Eq. 3 in Morrison and Milbrandt (2015).
 """
 function DSD_μ(p3::PSP3, λ::FT) where {FT}
-    @assert λ > FT(0)
+    #@assert λ > FT(0)
```

Comment:
Should we have an assert like this somewhere in the P3 scheme code? (Maybe not necessarily here)

Reply:
Lets try with the gamma functions